RecoLocalCalo/HcalRecAlgos/src/HcalSimpleRecAlgo.cc

```diff
@@ -323,7 +278,7 @@ namespace HcalSimpleRecAlgoImpl {
     bool leakCorrApplied = false;
     float t0 =0, t2 =0;
     float time = -9999;
-    float m3_time = -9999;
+    //    float m3_time = -9999;
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant